### PR TITLE
Fix for Compilation Error When Using FMT_MODULE

### PR DESCRIPTION
**Pull request by MathewBensonCode**

### CODE REVIEW

File: include/fmt/core.h [hunk 1]
```diff
@@ -1662,7 +1662,7 @@ template <typename Context> class basic_format_arg {
   \endrst
  */
 // DEPRECATED!
-FMT_EXPORT
+//FMT_EXPORT
```
Comment (vitaut) [inline include/fmt/core.h:1665]: Why comment out this?

**Review by vitaut — CHANGES_REQUESTED** [review]
Thanks for the PR. Please rebase it and resolve conflicts.

File: include/fmt/core.h (hunk 1 above)
Comment (MathewBensonCode) [reply include/fmt/core.h:1665]: I had commented it out when trying to diagnose the issue which was that there was an FMT_EXPORT  between a block of FMT_BEGIN_EXPORT and FMT_END_EXPORT. Forgot to take it out.

Now that it I managed to get it working I will take it out, rebase and resolve conflicts as requested.

Comment (vitaut) [reply include/fmt/core.h:1665]: Yeah, please remove this redundant `FMT_EXPORT` instead of commenting out.

Comment (MathewBensonCode) [reply include/fmt/core.h]: My apologies, I thought I had removed it in previous commit.

**Review by vitaut — APPROVED** [review]

**Comment by vitaut** [comment]
Merged, thanks.

**Pull request merged**